src/pages/index.tsx

```diff
@@ -94,7 +94,7 @@ export const getStaticProps: GetStaticProps = async ({ params }) => {
           comparison: 'gt'
         }, {
           field: 'density',
-          num: 0.005,
+          num: 0.5,
```

Comment:
Just my opinion but we were getting a lot of states on the explore page which I think is too broad to be of interest.

Reply:
fine with me